kubevirt-cdi/intro.md

```diff
@@ -0,0 +1,3 @@
+CDI is an utility designed to import Virtual Machine images for use with Kubevirt.
```

Comment:
Containerized Data Importer instead of CDI, the user just landed in the scenario and doesn't know what CDI means.